src/pages/register.tsx

```diff
@@ -72,15 +68,6 @@ export const RegisterPage: NextPage = () => {
   const onPasswordChange = useOnInputChange(setPassword)
   const onPasswordAgainChange = useOnInputChange(setPasswordAgain)
 
-  const router = useRouter()
-  useEffect(() => {
-    if (!allowRegister) {
-      router.push('/login').catch((reason) => logger.error('Error while redirecting to /login', reason))
-    } else if (userExists) {
-      router.push('/intro').catch((reason) => logger.error('Error while redirecting to /intro', reason))
-    }
-  }, [allowRegister, router, userExists])
-
```

Comment:
What is the problem here? Why was this removed? How should the frontend now handle a call to the /register if that's not allowed?

Reply:
![image](https://user-images.githubusercontent.com/6124140/185787782-5cc26ede-2b58-4dc6-9dd0-7fc07503faa4.png)

Reply:
![image](https://user-images.githubusercontent.com/6124140/185787798-bff84be7-8037-493e-9e17-193c1ba0ba9e.png)